### ISSUE TITLE
Adding new column in POCO does not show up in LAW

### ISSUE THREAD

**Issue by jusso-dev** [Zimmergren/LogAnalytics.Client#50]
Hey there,

Great library, love your work on this!

I have an issue when I attempt to add a new property to the .NET model I send to the target LAW, I input a new string property but it never shows up in the LAW table.. do I need to do anything for this to reflect in the LAW? do I need to manually update the columns in the table on the LAW?

Hope this makes sense.

**Comment by Zimmergren** [comment]
Hey @jusso-dev ,

You should be able to adjust your model and send new objects over without any manual tweaks. 
Can you reproduce the issue, or query LAW with KQL to see any new objects and what properties they have? 

However, it is likely that any new object contains the new properties, but older entries do not. So, if you make a query to retrieve items, but none of the added items are part of the result set, you may not see the column show up. 

Let me know if you've got a scenario to reproduce, or what your queries and data in LAW looks like.

**Comment by jusso-dev** [comment]
Thanks for getting back to me. 

If from your side you add an additional field to your existing class, does it show up in the LAW? All I did was add an additional string property to my class model and it does not show up in the LAW. My KQL is simply "table-name_s" | order by TimeGenerated desc

I've also left it 24 hours and nothing changed. 

I can't share anything specific as this is for a Government client of mine.

**Comment by Zimmergren** [comment]
I just did this, and it works fine on a demo project I've got lying around: 

1. Modify my .NET object to introduce a new property: 
![image](https://user-images.githubusercontent.com/4071728/223665775-0e793540-f26a-4d27-b67f-2444b303684c.png)

2. Set a value for this property in my entity before I send it off with the LogAnalytics client:
![image](https://user-images.githubusercontent.com/4071728/223665870-1e8f1442-6da2-45dc-98e6-281210be2fd2.png)

3. Query for the result, which shows that this property was successfully added as a string in the LAW data set: 
![image](https://user-images.githubusercontent.com/4071728/223665983-aed5de28-ed00-4fa2-838d-c3c216749036.png)
